examples/guard_riverpod/README.md

```diff
@@ -0,0 +1,19 @@
+# Guard Riverpod Example
+
+Example using [riverpod](https://pub.dev/packages/riverpod) package, with the release
+of `1.0.0-dev.7`, which drastically changes how things are done with this package
```

Comment:
It's a bit vague to what package is "which drastically changes how things are done with this package" referring to, as both riverpod introduced this major change and we have to handle it "unconventionally" in beamer.

Reply:
I was referring that `riverpod` drastically changed, not to beamer itself, as a comparison to previous versions. I will update this description.